scripts/buildAIProxyJsonSchema.ts

```diff
@@ -0,0 +1,40 @@
+/**
+ * Convert io-ts codec for transcendAIProxyPolicy.yml to a JSON Schema
+ *
+ * The output file should be unchanged across major versions.
```

Comment:
Yikes. That seems like a tall order for such a new product. 😂 Every time we add a route or policy, this is going to change.

Reply:
Ya I don't think this comment is relevant, so I'm removing it.